src/SFA.DAS.QnA.Api.Views/Views/QuestionPartials/_MonthAndYear.cshtml

```diff
@@ -16,7 +16,7 @@
 
 <div class="govuk-date-input" id="dob">
```

Comment:
To me it looks like it is missing this. I would check it highlights correctly in the error state:

<sfa-validationerror-detail sfa-validation-for="@(Model.Id)" sfa-validationerror-class="govuk-error-message"></sfa-validationerror-detail>